src/openforms/registrations/contrib/objects_api/submission_registration.py

```diff
@@ -336,11 +336,6 @@ def get_object_data(
                 "pdf_url": registration_data.pdf_url,
                 "csv_url": registration_data.csv_url,
             },
-            "auth_context": (
-                submission.auth_info.to_auth_context_data()
-                if submission.is_authenticated
-                else None
-            ),
```

Comment:
Just for my understanding, this is no longer needed because the `auth_context` variable is always available for the registration backend now?

Reply:
yeah, precisely. I assumed wrongly that the `auth_context` should be provided only as objects registration variables, but it acts at the same level as the existing `auth` static variable, so we can expose it directly to all registration backends automatically :)